source/fundamentals/bson.txt

```diff
@@ -55,15 +54,15 @@ Struct Tags
 -----------
 
 In Go, a **struct** is a collection of data fields with declared data
-types. The Go Driver can marshal/unmarshal structs and other native Go
-types to/from BSON using a `configurable codec system <{+api+}/bson/bsoncodec>`_.
+types. The {+driver-short+} can marshal and unmarshal structs and other native Go
+types to and from BSON by using a `configurable codec system <{+api+}/bson/bsoncodec>`_.
 
-You can modify the default marshalling and unmarshalling behavior of the Go Driver using
-**struct tags**, which are optional pieces of metadata attached to
-struct fields. The most common use of struct tags is for specifying the
-field name in the BSON document that corresponds to the struct field.
+You can modify the default marshalling and unmarshalling behavior of the
+{+driver-short+} by using **struct tags**, which are optional pieces of metadata
```

Comment:
This kinda makes it sound like the only way to modify BSON marshaling and unmarshaling behavior is using struct tags. However, users can also use `BSONOptions` to modify default marshaling/unmarshaling behavior. Generally struct tags are for modifying the marshaling/unmarshaling behavior of a single struct field and `BSONOptions` are for modifying the marshaling/unmarshaling behavior of an entire `Client`. Is there a way to clarify that?

Reply:
Edited to clarify this (both here and in the BSON options section)